### DISCUSSION TITLE
Using Navigator in Compose functions

### DISCUSSION THREAD

**Discussion by ladiesman2127** [readium/kotlin-toolkit#462] [Q&A] [answered]
Is that possible to use Readium in 1 activity model app? How can i use Navigator fragment in Compose Functions, and how can i make each word in publication clickable? Thank for the help!

**Comment by mickael-menu** [comment, marked as answer]
It should be possible to use the navigators with Jetpack Compose with the [interoperability APIs](https://developer.android.com/jetpack/compose/migrate/interoperability-apis/views-in-compose).

As for making each word clickable, it's not a feature of the toolkit. However you could inject your own JavaScript files using: https://github.com/readium/kotlin-toolkit/discussions/260#discussioncomment-3349219

and the JavaScript interface APIs: https://github.com/readium/kotlin-toolkit/blob/develop/CHANGELOG.md#navigator-7
